.travis.yml

```diff
@@ -13,8 +13,6 @@ matrix:
       env: TOXENV=jessie
     - python: 2.7
       env: TOXENV=pypy
-    - python: 3.3
-      env: TOXENV=py33
```

Comment:
not tested under python3.4?

Reply:
I've added it in a new commit.

In https://github.com/scrapy/scrapy/pull/1674#issuecomment-171288714 they decided not to test on 3.4, let me know if you prefer it left out.

Reply:
by that time 3.5 was the latest release on 3.x branch and 3.3 was the smaller version we supported by then.

Looks like the unwritten policy was to test for the minimum version and the latest, what was 3.3. and 3.5 then now it is 3.4 and 3.6. 

Anyway, I'd left 3.5 until we realize it is taking too long to run travis-ci builds, or simply until 3.7 get released.